src/redux/actions/dictionaries/dictionaryActionCreators.js

```diff
@@ -144,5 +146,33 @@ export const releaseHead = (url, data) => (dispatch) => {
       .catch(error => {
         notify.show(`${error.response.data.__all__[0]}`, 'error', 6000);
       })
-    }
+};
+
+export const createVersion = (url, data) => (dispatch) => {
+  return api.dictionaries
```

Comment:
You could remove the body or brackets and remove the return statement.

Reply:
Yes that is possible but would it not look odd, since all other functions on the file use the above convention?